src/design-system/InputText.tsx

```diff
@@ -44,6 +43,17 @@ const WrappedInput = styled(CustomDebounceInput)`
   }
 `;
 
+const InlineIcon = styled.img<{ isReference?: boolean }>`
```

Comment:
don't need to define this prop anymore

Reply:
you beat me to it! just pushed this change